internal/pkg/deploy/cloudformation/stack/validate_template_integration_test.go

```diff
@@ -1,4 +1,6 @@
+//go:build integration
```

Comment:
oh why did we add this?

Reply:
shoot. it's added by my ide whenever i save the file. not only for vscode but sublime as well

Reply:
np! we can remove it in a separate PR :D